packages/graphback-datasync/src/providers/createDataSyncMongoDbProvider.ts

```diff
@@ -1,20 +1,31 @@
 import { Db } from 'mongodb';
-import { ModelDefinition } from '@graphback/core';
+import { ModelDefinition, defaultTableNameTransform } from '@graphback/core';
 import { MongoDBDataProvider, GraphbackDataProvider } from "@graphback/runtime-mongo"
 import { isDataSyncModel } from '../util';
+import { PruneScheduler, PruneConfiguration, TTLMap } from "../scheduler";
 import { DataSyncMongoDBDataProvider } from './DatasyncMongoDBDataProvider';
 
 /**
  * Creates a new Data synchronisation data provider for MongoDb
  *
  * @param {Db} db - MongoDb connection
  */
-export function createDataSyncMongoDbProvider(db: Db): (...args: any[]) => GraphbackDataProvider {
+export function createDataSyncMongoDbProvider(db: Db, ttlMap?:TTLMap, pruneConfig: PruneConfiguration = {}): (...args: any[]) => GraphbackDataProvider {
```

Comment:
Instead of map I will carry this information in model.
Benefit is that we can expose this later in schema or something or have global TTL for the moment